webapp/src/main/resources/public/js/components/drops/Drops.js

```diff
@@ -276,7 +277,7 @@ let Drops = createReactClass({
 
         return (
             <tr key={`Drops.repositoryRow.${drop.name}.${drop.repository.name}`} className={rowClass}>
-                <td>{drop.name}{this.getButtonControlBar(drop)}</td>
+                <td>{drop.name}{AuthorityService.canEditProjectRequests() && this.getButtonControlBar(drop)}</td>
```

Comment:
also see later comment. Here i'm fine calling the AuthorityService directly because there is already some state in the component.

For stateless component, I'd rather avoid though